src/patternfly/components/Table/table.scss

```diff
@@ -154,7 +154,7 @@
   background-color: var(--pf-c-table--BackgroundColor);
 
   td {
```

Comment:
Can we set this on all cells, so that we can cover `tbody > tr > th`? If we continue to use `white-space: nowrap` on `thead`, that should disable this behavior.

Reply:
yep